lib/public/Model.js

```diff
@@ -252,29 +252,13 @@ export default class Model extends Observable {
     }
 
     /**
-     * Toggle navbar overview dropdown
-     * @returns {undefined}
-     */
-    toggleOverviewDropdown() {
-        this.dropdownMenu = this.dropdownMenu === 'overview' ? false : 'overview';
-        this.notify();
-    }
-
-    /**
-     * Toggle navbar account dropdown
-     * @returns {undefined}
-     */
-    toggleAccountDropdown() {
-        this.dropdownMenu = this.dropdownMenu === 'account' ? false : 'account';
-        this.notify();
-    }
-
-    /**
-     * Clears all navbar dropdowns
-     * @returns {undefined}
+     * Toggle navbar dropdown
+     * @param {string|null} key referencing dropdown to be toggled,
+     * default value is null, which causes closing all dropdowns
+     * @returns {void}
      */
-    clearDropdownMenu() {
-        this.dropdownMenu = false;
+    toggleDropdownMenu(key = null) {
```

Comment:
No, toggle is different from clear. Please keep the clear function separated, and do not allow for null key.

Reply:
clear function restored